assets/js/googlesitekit/datastore/user/dismissed-items.js

```diff
@@ -154,6 +165,28 @@ const baseActions = {
 			);
 		}
 	),
+	setItemDimissingState( slug, state ) {
+		return {
+			payload: { slug, state },
+			type: 'SET_ITEM_DISMISSING_STATE',
+		};
+	},
+};
+
+const baseReducer = ( state, { type, payload } ) => {
+	switch ( type ) {
+		case 'SET_ITEM_DISMISSING_STATE':
+			const { slug, state: isDismissing } = payload;
```

Comment:
I realise this is following the approach outlined in the IB, but using "state" in the names here is really quite confusing, as "state" is such a fundamental Redux concept. Passing an argument called `state` to an action makes it look like a selector at first glance.

Rather than overloading this term I'd suggest we make the following changes:

```suggestion
	setIsItemDimissing( slug, isDismissing ) {
		return {
			payload: { slug, isDismissing },
			type: 'SET_IS_ITEM_DISMISSING',
		};
	},
};

const baseReducer = ( state, { type, payload } ) => {
	switch ( type ) {
		case 'SET_IS_ITEM_DISMISSING':
			const { slug, isDismissing } = payload;
```